streams/src/main/java/org/apache/kafka/streams/processor/internals/SinkNode.java

```diff
@@ -97,6 +96,10 @@ public void process(final K key, final V value) {
                                     keyClass,
                                     valueClass),
                     e);
+        } catch (final StreamsException e) {
```

Comment:
Hmm.. we did not override the `toString` function of `StreamsException`, I'm wondering what would be printed in `%s` for the exception `e`?